src/components/Accordion/Accordion.test.tsx

```diff
@@ -47,14 +48,14 @@ describe("Accordion", () => {
         titleElement="h4"
       />
     );
-    expect(wrapper.find("AccordionSection").at(0).prop("titleElement")).toBe(
-      "h4"
-    );
+    // Query for the specific element as defined in the titleElement prop.
+    // eslint-disable-next-line testing-library/no-node-access
+    expect(document.querySelector("li h4")).toBeInTheDocument();
```

Comment:
"li h4" selector seems to be a bit too dependant on the HTML structure, and doesn't truly test what we want to test here (that the title element is h4).

Ideally we should get the title element using its role or text (we know the title) and check if it is an h4.

Reply:
Done. Does that look better?